### DISCUSSION TITLE
How to enable top navigation in v2 ?

### DISCUSSION THREAD

**Discussion by monzer15** [filamentphp/filament#8799] [Help] [answered]
How to enable top navigation in version 2 ?

**Comment by VincvanAs** [comment]
What do you mean by top navigation? You can render hooks and create your own. But there is no such thing as enabling the top navigation.

https://filamentphp.com/docs/2.x/admin/appearance

**Reply by monzer15** [reply]
for example, in v3 there is a function in the panel builder called topNavigation(), my question is how to achieve the same behavior in v2, thank u.

**Reply by leandrocfe** [reply, marked as answer]
topNavigation is a feature from v3. There isn't on v2
